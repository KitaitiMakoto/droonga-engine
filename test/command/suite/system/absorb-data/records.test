#@require-catalog-version 2
#@disable-logging
{
  "type": "add",
  "dataset": "Source",
  "body": {
    "table": "Users",
    "key": "Adam"
  }
}
#@enable-logging
{
  "type": "system.absorb-data",
  "dataset": "Default",
  "body": {
    "host":    "localhost",
    "port":    23003,
    "tag":     "droonga",
    "dataset": "Source"
  }
}
