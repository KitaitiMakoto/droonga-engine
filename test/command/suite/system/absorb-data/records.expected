{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "type": "system.absorb-data.start"
}
{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "system.absorb-data.result",
  "body": {
    "started": true
  }
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "body": {
    "nProcessedMessages": 3,
    "nRestoredObjects":   3,
    "percentage":         100,
    "message":            "100% done (maybe 00:00:00 remaining)"
  },
  "type": "system.absorb-data.progress"
}
{
  "inReplyTo": "request-id",
  "dataset": "Default",
  "date": "0000-00-00T00:00:00.000000Z",
  "type": "system.absorb-data.end"
}
